backend/apps/volontulo/views/api.py

```diff
@@ -65,6 +69,62 @@ def login_view(request):
     )
 
 
+@api_view(['POST'])
+@authentication_classes((CsrfExemptSessionAuthentication,))
+@permission_classes((AllowAny,))
+def register_view(request):
+    """REST API register view."""
+    if not request.user.is_authenticated():
+        email = request.data.get('email')
```

Comment:
Instead of nesting all the function body one level to the right, you can check and handle the corner case condition at the beginning of the function, and then process with its execution. It's easier to stay within 80 line chars limit and it's just more readable. Sth like: 
```
if request.user.is_authenticated():
    return 400 something

here all your "proper" logic.
```

Reply:
+1